src/main/java/pl/pszczolkowski/mustdo/domain/task/bo/TasksListBO.java

```diff
@@ -0,0 +1,15 @@
+package pl.pszczolkowski.mustdo.domain.task.bo;
+
+import pl.pszczolkowski.mustdo.domain.task.dto.TasksListSnapshot;
+
+public interface TasksListBO {
+
+   TasksListSnapshot add(String name, Long boardId);
+
+   TasksListSnapshot rename(Long id, String name);
```

Comment:
When quoting my words, try to quote them more accurately. Anyway change this method so it won't return any snapshot